src/settings/StaffSlips/StaffSlipDetail.js

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import HtmlToReact, { Parser } from 'html-to-react';
-import { FormattedMessage } from 'react-intl';
+import {
+  FormattedMessage,
```

Comment:
Could you please check if we can replace all `FormattedMessage` with `formatMessage` and drop this import at all?